components/Downloads/ChangelogModal/index.module.css

```diff
@@ -0,0 +1,98 @@
+.overlay {
+  @apply fixed
+    inset-0
+    flex
+    justify-center
+    bg-white
+    bg-opacity-90
+    backdrop-blur-lg
+    dark:bg-neutral-950
+    dark:bg-opacity-80;
+
+  .content {
+    @apply relative
+      mx-4
+      my-4
+      inline-flex
+      w-full
+      flex-col
+      overflow-scroll
+      rounded
+      border
+      border-neutral-200
+      bg-white
+      p-8
+      focus:outline-none
+      dark:bg-neutral-950
+      sm:mt-20
+      sm:w-1/2
+      sm:p-12;
+  }
+
+  .trigger {
+    @apply absolute
```

Comment:
Should we really style the trigger? Knowing that the trigger can be anything? A simple a href, a button element or our button component?

Reply:
(What are these styles even for? 🤔)

Reply:
This is just the close button I added for mobile design, I realized there was nothing for the user to close this modal so I added it 😄

Reply:
Might be worth calling it "closeTrigger" or "close", as "trigger" is the prop for the button that opens the modal xD